airbyte-config/init/copy_seed_data.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
needs to be in a script because the parsers freaks out if it sees a `$` that isn't use for variable interpolation.